### DISCUSSION TITLE
Suggest an idea for DeFiChain

### DISCUSSION THREAD

**Discussion by izzycsy** [DeFiCh/app#677] [Ideas]
#### Is your feature request related to a problem? Please describe.
_A clear and concise description of what the problem is. Eg. I'm always frustrated when [...]_

#### Describe the solution you'd like
_A clear and concise description of what you want to happen._

#### Describe alternatives you've considered
_A clear and concise description of any alternative solutions or features you've considered._ 

#### Additional context
_Add any other context or screenshots about the feature request here._

**Comment by DirkSchumacher0** [comment]
Hi Guys..

I hope You doing Well. Is there any possibility to implemented a Two-factor authentication at the DefiChain Wallet.
The same like we are using for cake.

I'm very proud to be a part of this nice community and project. 
All the best Dirk

**Comment by AndiyNW** [comment]
💡 Configuration of UserInterface / Feature

Hi

it would be great, if we would have in the setup/configuration of the wallet an option to configure how the user interface looks like, like sorting of the displayed tokens, priorities of tokens displayed, or like display last setup after exit and restart. Or the last address used as preconfigured selection and such.

I think, it would be nice and just an add-on, though I do know, priorities are currently on other projects

thanks
ciao
Andiy

**Comment by AndiyNW** [comment]
💡 External Link/Reference Input for Display / Feature

would be cool, if we could configure external resources for references, like configurable Centralised exchanges as input to the current value of BTC there or $DFI from the current available CEXs? could be done via linking via APIs I would assume or from CoinMarketCap and alike? So people have the option, to have a display of current US$ or € or BTC value of their holdings?

Would be a nice to have functionality.

thanks
ciao
Andiy

**Comment by mirbach** [comment]
Hey guys, I would like to see some hardware wallet support for trezor and ledger. Are there any timelines you can share?

**Comment by jannn48** [comment]
Hi,

it would be great if a Cyber Security ETF could be added to the list for the coming dToken voting.

One example would be: L&G Cyber Security UCITS ETF

Best,
Jan
